src/components/WorldListPage/WorldContributorTab/WorldContributorTab.tsx

```diff
@@ -0,0 +1,99 @@
+import { t } from 'decentraland-dapps/dist/modules/translation'
+import { Loader, Message, Table, Empty, Button } from 'decentraland-ui'
+import { formatNumber } from 'decentraland-dapps/dist/lib'
+import { useCallback } from 'react'
+import { config } from 'config'
+import { locations } from 'routing/locations'
+import Profile from 'components/Profile'
+import { ENS } from 'modules/ens/types'
+import { fromBytesToMegabytes, renderPublishSceneButton, renderWorldUrl } from '../utils'
+import { Props } from './WorldContributorTab.types'
+
+export default function WorldContributorTab({ items, deploymentsByWorlds, projects, loading, error, onNavigate, onUnpublishWorld }: Props) {
+  const handlePublishScene = useCallback(() => {
+    onNavigate(locations.scenes())
+  }, [onNavigate])
+
+  const handleEditScene = useCallback(
+    (ens: ENS) => {
+      const { projectId } = deploymentsByWorlds[ens.subdomain]
+      onNavigate(locations.sceneDetail(projectId as string))
+    },
+    [deploymentsByWorlds, locations, onNavigate]
+  )
+
+  const handleUnpublishScene = useCallback(
+    (ens: ENS) => {
+      const deploymentId = deploymentsByWorlds[ens.subdomain]?.id
+      if (deploymentId) {
+        onUnpublishWorld(deploymentId)
+      }
+    },
+    [deploymentsByWorlds, onUnpublishWorld]
+  )
+
+  if (error) {
+    return <Message error size="tiny" visible content={error.message} header={t('worlds_list_page.error_title')} />
+  }
+
+  if (loading) {
+    return <Loader active size="big" />
+  }
+
+  if (!items.length) {
+    return (
+      <Empty className="empty-names-container" height={500}>
```

Comment:
This component has classes, but it doesn't have a `module.css` associated with it, is it getting styled by a component which is up the chain?

Reply:
yes its getting it from the parent component